api/src/adapters/search/opensearch_client.py

```diff
@@ -175,8 +180,15 @@ def swap_alias_index(
         self._client.indices.update_aliases({"actions": actions})
 
         # Cleanup old indexes now that they aren't connected to the alias
+        resp = self._client.cat.indices(format="json", h=["index"])
+        old_indexes = [
+            index["index"]
+            for index in resp
+            if index["index"].startswith(index_prefix) and index["index"] != index_name
+        ]  # omit the newly created one
+
```

Comment:
I think you can simplify this a bit, when querying OpenSearch you can specify a wildcard:

```
GET _cat/indices/opportunity-index-*
```

Should be able to do that here as well?